File: Java-Core/src/main/java/presto/python/PythonModule.java
package presto.python;

import java.util.Collection;

import presto.utils.CodeWriter;

public class PythonModule {

	Collection<String> identifiers;
	
	public PythonModule(Collection<String> identifiers) {
		this.identifiers = identifiers;
	}

	public void toDialect(CodeWriter writer) {
		writer.append(" from module: ");
		for(String id : identifiers) {
			writer.append(id);
			writer.append('.');
		}
		writer.trimLast(1);
	}

}

File: Java-CodeServer/src/main/resources/js/prompto/python/PythonModule.js
function PythonModule(ids) {
    this.ids = ids;
}

PythonModule.prototype.toDialect = function(writer) {
    writer.append(" from module: ");
    for(var i=0;i<this.ids.length;i++) {
        writer.append(this.ids[i]);
        writer.append('.');
    }
    writer.trimLast(1);
}

exports.PythonModule = PythonModule;